packages/client/src/lobby/createReplayJSON.ts

```diff
@@ -63,9 +64,10 @@ export default function createJSONFromReplay(room: string) {
     () => {},
```

Comment:
get rid of the then crap

Reply:
need to accept either this or #2574 first so I known where to merge from

Reply:
up to you, which one do you want to focus on first

Reply:
accept the other one first please.